config/base/params.env

```diff
@@ -4,4 +4,4 @@ caikit-standalone-image=quay.io/opendatahub/caikit-nlp:fast
 tgis-image=quay.io/opendatahub/text-generation-inference:fast
 ovms-image=quay.io/opendatahub/openvino_model_server:2024.3-release
 vllm-image=quay.io/opendatahub/vllm:fast
-nim-state=removed
+nim-state=managed
```

Comment:
is this expected to be enabled by default?

Reply:
@spolti yes... Please reference --> https://redhat-internal.slack.com/archives/C07CZDA6DE1/p1734707472304319?thread_ts=1734545514.363599&cid=C07CZDA6DE1